mava/logger.py

```diff
@@ -102,19 +115,52 @@ def log(
                 f"Entropy {float(np.mean(entropy)):.3f}"
             )
             logger.console_logger.info(
-                f"{Fore.MAGENTA}{Style.BRIGHT}TRAINER: {log_string}{Style.RESET_ALL}"
+                f"{Fore.MAGENTA}{Style.BRIGHT}TRAINER{suffix}: {log_string}{Style.RESET_ALL}"
             )
         else:
             logger.console_logger.info(
-                f"{Fore.GREEN}{Style.BRIGHT}EVALUATOR: {log_string}{Style.RESET_ALL}"
+                f"{Fore.GREEN}{Style.BRIGHT}EVALUATOR{suffix}: {log_string}{Style.RESET_ALL}"
             )
 
         return float(np.mean(episodes_return))
 
-    return log
+    def save_json_by_seed() -> None:
+        """Split the current json file by seed into separate files."""
+        # Make sure the full values are recorded in the JSON file.
+        time.sleep(10)
```

Comment:
I think this should rather be pulled out of this function otherwise people might try to call this function and expect it to happen instantly. So rather

```python
# Make sure the full values are recorded in the JSON file.
time.sleep(10)
logger.save_json_by_seed()
```

Reply:
the issue of moving it outside will need to check with the flag that we will call the function else will be useless 10 seconds of waiting do you have another suggestion? else maybe we can just shows a print related to it "waiting for 10 seconds..."

Reply:
oh I saw the other review point yeah we can add time.sleep in the stop function